components/partial/MDXStyleWrapper.tsx

```diff
@@ -99,6 +99,11 @@ const MDXStyleWrapper = ({ children }: { children: any }) => (
           ${codeStyles.styles}
         }
       }
+
+      > div {
```

Comment:
I ended up just adding some default margin to root `div` elements within MDXWrapper.